src/media/js/previews.js

```diff
@@ -233,11 +233,18 @@ define('previews',
     z.page.on('dragstart', '.previews-tray', utils._pd)
 
     .on('click', '.previews-tray ' + previewItemClass, function(e) {
-        // If a tray thumbnail is clicked, open lightbox.
-        // Don't show on desktop detail since it'd show the previews at
-        // the same size.
-        if (!isDesktopDetail()) {
+        var isVideo = e.target.nodeName === 'VIDEO';
```

Comment:
would probably move all the isVideo logic into one block

if (!isVideo || !isDesktopDetail() && isVideo) {
e.preventDefault();
}